src/components/PassphraseDialog/PassphraseDialog.scss

```diff
@@ -21,12 +23,12 @@ $color-backlog-blue--dark: #5c8fff;
 
 .passphrase-dialog__form {
   background-color: $color-white;
+  color: $color-dark-mode-note;
   border-radius: 24px;
   box-shadow: 0px 4px 15px 0px rgba($color: $color-backlog-blue--light, $alpha: 0.25);
   margin: 0 auto;
 
   padding: $spacing--xl $spacing--lg;
-  gap: $spacing--base;
```

Comment:
Because otherwise the gap is also there between the wrong-password hint and the password input field, where it should not be. I didn't find a way to set the gap individually for different rows/columns